webapp/src/components/tutorial/TutorialContainer.tsx

```diff
@@ -110,17 +110,14 @@ export function TutorialContainer(props: TutorialContainerProps) {
     const hintMarkdown = steps[visibleStep].hintContentMd;
 
     const validateTutorialStep = async () => {
-        const globalValidators =
-            tutorialOptions.globalValidationConfig?.validators ??
-            PopulateValidatorCache(tutorialOptions.globalValidationConfig);
-        const localValidators =
-            currentStepInfo.localValidationConfig?.validators ??
-            PopulateValidatorCache(currentStepInfo.localValidationConfig);
-
-        let validators: pxt.Map<CodeValidator> = {
-            ...(globalValidators ?? {}),
-            ...(localValidators ?? {})
-        };
+        let validators: pxt.Map<CodeValidator> = {};
+
```

Comment:
If there's a more elegant way to populate `validators` here, I'm open to suggestions. I figured this way, we at least don't unnecessarily create global validators that are just going to be replaced by local ones, but the code itself seems a little messy.

Reply:
Eh, I think it's fine here, think the alternatives are typically pretty ugly and it's mapping an array to an object anyways.

If you really wanted you could do something like `(currentStepInfo.localValidationConfig.validatorsMeta ?? []).concat(tutorialOptions.globalValidationConfig ?? [])` and keep the same `if (!validators[blah])`, and possible use an `Array.reduce`, but neither of those typically end up particularly pretty anyways

Reply:
Hmm fair point. Suppose I'll leave it as-is for now, then. Thanks for the info!